semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/rpc/RpcSemanticConventionUtils.java

```diff
@@ -0,0 +1,65 @@
+package org.hypertrace.semantic.convention.utils.rpc;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import java.util.List;
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.shared.SpanAttributeUtils;
+import org.hypertrace.core.span.constants.RawSpanConstants;
+import org.hypertrace.core.span.constants.v1.CensusResponse;
+import org.hypertrace.core.span.constants.v1.Grpc;
+import org.hypertrace.semantic.convention.utils.span.SpanSemanticConventionUtils;
+
+/**
+ * Utility class to fetch rpc attributes
+ *
+ * The methods in this class might work under the
+ * assumption that span data is for specific rpc system
+ */
+public class RpcSemanticConventionUtils {
+
+  // otel specific attributes
+  private static final String OTEL_RPC_SYSTEM = OTelRpcSemanticConventions.RPC_SYSTEM.getValue();
+  private static final String OTEL_RPC_METHOD = OTelRpcSemanticConventions.RPC_METHOD.getValue();
+  private static final String OTEL_GRPC_STATUS_CODE = OTelRpcSemanticConventions.GRPC_STATUS_CODE.getValue();
+  private static final String OTEL_RPC_SYSTEM_GRPC = OTelRpcSemanticConventions.RPC_SYSTEM_VALUE_GRPC.getValue();
+
+  private static final String OTHER_GRPC_HOST_PORT = RawSpanConstants.getValue(Grpc.GRPC_HOST_PORT);
+  private static final String OTHER_GRPC_METHOD = RawSpanConstants.getValue(Grpc.GRPC_METHOD);
+  private static final String[] OTHER_GRPC_STATUS_CODES =
+      {
+          RawSpanConstants.getValue(CensusResponse.CENSUS_RESPONSE_STATUS_CODE),
+          RawSpanConstants.getValue(Grpc.GRPC_STATUS_CODE),
+          RawSpanConstants.getValue(CensusResponse.CENSUS_RESPONSE_CENSUS_STATUS_CODE)
+      };
+
+  /**
+   * @return attribute keys for grpc method
+   */
+  public static List<String> getAttributeKeysForGrpcMethod() {
+    return Lists.newArrayList(Sets.newHashSet(OTEL_RPC_METHOD, OTHER_GRPC_METHOD));
```

Comment:
In most of the places the order is `other` first and then `otel`, can we have the same here too?